codespell_lib/data/dictionary.txt

```diff
@@ -18765,6 +18765,8 @@ soket->socket
 sokets->sockets
 solarmutx->solarmutex
 solatary->solitary
+solderd->soldered, solder, soldier, soldiered,
+soldert->soldered, solder, soldier, soldiered,
```

Comment:
I'm not convinced this one should suggest the ed variants.

Reply:
How much convincing do you need?
```
acccessd->accessed
acknowldegement->acknowled
adapd->adapt, adapted, adopt, adopted
addedd->added
alignd->aligned
allowd->allowed
anchord->anchored
antecedant->anteced
appendent->appended
aquaduct->aqued
assigend->assigned
assignend->assigned
assumad->assumed
assumud->assumed
blessd->blessed
bord->board, bored
bufferent->buffered
calld->called
capturd->captured
carreid->carried
centerd->centered
checkd->checked
chekd->checked
cloesd->closed
combind->combined
commenetd->commented
commpresd->compressed
commpressd->compressed
configuerd->configured
conflictd->conflicts, conflicted
conquerd->conquered
considerd->considered
controllerd->controlled
correlatd->correlated
createdd->created
datecreatedd->datecreated
decendend->descendent, descendant, descended
decompressd->decompressed
decremenetd->decremented
decrmenetd->decremented
defiend->defined
defind->defined
define'd->defined, disabled
definend->defined
dependend->dependent, depended
descendend->descended
designd->designed
detaild->detailed
detectetd->detected
determind->determined
directd->directed
disapeard->disappeared
disobediant->disobed
displayd->displayed
dissapeard->disappeared
dissobediant->disobed
dissobedient->disobed
dont->don't, disabled
droppend->dropped
duplicatd->duplicated
earnt->earned
editedt->edited
editt->ed
embedd->embed
enameld->enameled
encounterd->encountered
enhancd->enhanced
estabishd->established
etablishd->established
existend->existed
expectd->expected
faield->failed
faild->failed
failt->fail, failed
flattend->flattened
follod->followed
followd->followed
formend->formed
forst->first, forced
groupd->grouped
groupt->grouped
gruopd->grouped
guarenteerd->guaranteed
guarentyd->guaranteed
guarrantyd->guaranteed
handeld->handled
hapend->happened
happend->happened
hunderd->hundred
ignord->ignored
ignorgd->ignored
ignormd->ignored
ignornd->ignored
ignorrd->ignored
ignortd->ignored
implemenetd->implemented
implmenetd->implemented
importd->imported
incremenetd->incremented
incrmenetd->incremented
initialisiert->initialised, initialized
instrumenetd->instrumented
interbread->interbreed, interbred
jagid->jagged
jumpt->jumped
labeld->labelled
lenghtend->lengthened
liekd->liked
manufacturedd->manufactured
mentiond->mentioned
mimachd->mismatched
mimatchd->mismatched
mismatchd->mismatched
missmatchd->mismatched
modifid->modified
mut->must, mutt, moot, disabled
namesd->named
nead->need
nedd->need
neeedn't->need
neesd->need
obediant->obed
obtaiend->obtained
occurd->occurred
offerd->offered
openend->opened
orderd->ordered
pre-defiend->pre-defined
precendent->preced
predefiend->predefined
preferend->preferred
pretendend->pretended
programmend->programmed
providewd->provided
publishd->published
pusehd->pushed
queueud->queued
raisedd->raised
re-defiend->re-defined
redandant->red
redefiend->redefined
redict->red
redirectd->redirected
redunant->red
redundent->red
reduntant->red
referenct->referenced
refererd->referred
reformattd->reformatted
registerd->registered
registeresd->registered
registert->registered
relatd->related
releasead->released
removerd->removed
representd->represented
requestd->requested
requestesd->requested
requierd->required
reserverd->reserved
resest->reset, recessed
reseverd->reserved
returend->returned
returnd->returned
scipt->script, skipped
secant->second, disabled
segmenetd->segmented
selecetd->selected
selectd->selected
shortend->shortened
skept->skipped
skipd->skipped
skippd->skipped
skipt->skip, Skype, skipped
softend->softened
solderd->soldered, solder, soldier, soldiered
soldert->soldered, solder, soldier, soldiered
sord->sword, sore, sored, sawed
soterd->stored, sorted
specifid->specified
startd->started
stript->stripped
subdivisiond->subdivisioned
succedd->succeed
superseedd->superseded
supportd->supported
talekd->talked
tesselatad->tessellated
threatend->threatened
transferd->transferred
transferrd->transferred
transfert->transfer, transferred
triggerd->triggered
uesd->used
uint->unit, disabled
uncommpressd->uncompressed
undefiend->undefined
unexpect->unexpected
unexpectend->unexpected
unmappend->unmapped
unorderd->unordered
unrecoverd->unrecovered
unregisterd->unregistered
unregistert->unregistered
unrelatd->unrelated
unresgisterd->unregistered
unsupport->unsupported
upporterd->supported
upstreamedd->upstreamed
user-defiend->user-defined
```

Reply:
(found some incorrect entries because of that, thanks :) )

Reply:
It was only the rt->ed ones I was meaning, which is only really `unsupport->unsupported` in the dictionary, which is a different kind of fix.

Reply:
> It was only the rt->ed ones I was meaning, which is only really `unsupport->unsupported` in the dictionary, which is a different kind of fix.  
It's a few more than just that one but it's less than what I posted before...  

```
appendent->appended
bufferent->buffered
earnt->earned
editedt->edited
groupt->grouped
referenct->referenced
registert->registered
skept->skipped
stript->stripped
unexpect->unexpected
unregistert->unregistered
unsupport->unsupported
```

Reply:
IMO, it's justified.

Reply:
We probably only want to add ones we actually see in the wild

Reply:
@larsoner https://github.com/search?q=solderd&type=Code

Reply:
This would be the proper search, right?

https://github.com/search?p=5&q=soldert&type=Code

I looked at two random pages from the results and they were all false alarms

Reply:
@larsoner I'll remove `soldert` then and keep `solderd`

Reply:
You haven't removed it yet @luzpaz .

Reply:
@larsoner https://www.google.com/search?q="soldert+a"&ie=UTF-8&oe=UTF-8

Reply:
94 results across all of Google is not very many. Not worth the false alarms that were showing up in the GitHub search

Reply:
Moved to rare dictionary: d3d47c2

Reply:
The rare dictionary is "for rare but valid words", I don't believe this word fits that definition:
https://github.com/codespell-project/codespell/blob/master/codespell_lib/_codespell.py#L45